app/views/posts/_post.html.erb

```diff
@@ -1,68 +1,72 @@
 <%= stylesheet_link_tag 'posts' %>
```

Comment:
Shouldn't it be wider on profiles? 

![image](https://user-images.githubusercontent.com/51726526/122298662-6a31c500-ced3-11eb-9ef4-3806ce1e0055.png)

Reply:
Fixed.